diff --git a/image-types/global.d.ts b/image-types/global.d.ts
index 661c316e89b3c1aa5913b8afebdbf3e6299eccd7..499a7fe5988eb1025bf24a87a178b3012c045df7 100644
--- a/image-types/global.d.ts
+++ b/image-types/global.d.ts
@@ -8,14 +8,9 @@ declare module '*.png' {
 }
 
 declare module '*.svg' {
-  /**
-   * Use `any` to avoid conflicts with
-   * `@svgr/webpack` plugin or
-   * `babel-plugin-inline-react-svg` plugin.
-   */
-  const content: any
+  const SVG: React.FC<React.SVGProps<SVGSVGElement>>
 
-  export default content
+  export default SVG
 }
 
 declare module '*.jpg' {
