diff --git a/image-types/global.d.ts b/image-types/global.d.ts
index 661c316e89b3c1aa5913b8afebdbf3e6299eccd7..6f99a6433269a88680587738b21e18b926e499a1 100644
--- a/image-types/global.d.ts
+++ b/image-types/global.d.ts
@@ -13,9 +13,9 @@ declare module '*.svg' {
    * `@svgr/webpack` plugin or
    * `babel-plugin-inline-react-svg` plugin.
    */
-  const content: any
+  const SVG: React.FC<React.SVGProps<SVGSVGElement>>
 
-  export default content
+  export default SVG
 }
 
 declare module '*.jpg' {
